test/micro/org/openjdk/bench/java/lang/foreign/CLayouts.java

```diff
@@ -64,7 +65,8 @@ public class CLayouts {
     /**
      * The layout for the {@code double} C type
      */
-    public static final ValueLayout.OfDouble C_DOUBLE = ValueLayout.JAVA_DOUBLE;
+    public static final ValueLayout.OfDouble C_DOUBLE = ValueLayout.JAVA_DOUBLE.withByteAlignment(IS_AIX ? 4 : 8);
```

Comment:
Would prefer if this used `Linker.canonicalLayouts`
```suggestion
    public static final ValueLayout.OfDouble C_DOUBLE = (ValueLayout.OfDouble) LINKER.canonicalLayouts().get("double");
```

Reply:
I've improved the micro benchmarks with 3rd commit.